packages/datadog-plugin-next/src/index.js

```diff
@@ -73,14 +90,21 @@ class NextPlugin extends ServerPlugin {
     const span = store.span
     const req = this._requests.get(span)
 
+    // safeguard against missing req in complicated timeout scenarios
+    if (!req) return
+
+    const errorPages = ['/404', '/500', '/_error', '/_not-found']
+
     // Only use error page names if there's not already a name
     const current = span.context()._tags['next.page']
-    if (current && ['/404', '/500', '/_error', '/_not-found'].includes(page)) {
+    if (current && errorPages.includes(page)) {
```

Comment:
Let's cache the result of `errorPages.includes(page)` somewhere so that we don't have to call it twice. Might be a slight perf bump?